tests/test_transpilers_routing.py

```diff
@@ -0,0 +1,239 @@
+import networkx as nx
+import numpy as np
+import pytest
+from qibo import gates
+from qibo.models import Circuit
+
+from qibolab.transpilers.placer import Custom, Subgraph, Trivial, check_placement
+from qibolab.transpilers.routing import (
+    ShortestPaths,
+    remap_circuit,
+    respect_connectivity,
+)
+
+
+def star_connectivity():
+    Q = ["q" + str(i) for i in range(5)]
```

Comment:
Since you're not drawing it (these are just tests) do we really need this `Q`? (at this point, you know I don't like it xD)

Reply:
I decided to keep the strings in order to distinguish between physical (string) and logica (number) qubits.
This is necessary because developing the code i constantly need to print graphs and connectivity and differentiating between strings and int simplify a lot the work. when all the future code (sabre and backpropagation) will be ready we can think about using only numbers. however i think that also from the user perspective having a clear distinction between physical and logical qubits may be a good idea.

Reply:
yes but then i need to use these values in the nx.graphs and print them...
moreover in some nx functions the mapping is 'inverted' making it really difficult to work with without a clear distinction